docker-entrypoint.sh

```diff
@@ -2,29 +2,61 @@
 set -e
 
 export GC_DOMAIN="${RENDER_EXTERNAL_HOSTNAME}"
+export DB_HOSTNAME=${DB_HOSTNAME:-'oregon-postgres.render.com'}
```

Comment:
Is `DB_HOSTNAME` expected to be set? I'm not seeing it set anywhere, but I may be missing it. If it's not set, wouldn't this cause EU deployments to fail?

Reply:
It will be taken from the service env vars (user entry)